app/user/invite-users/invite-users.component.html

```diff
@@ -0,0 +1,21 @@
+<md-card class="default-card">
+  <h1>{{ 'invite users' | translate}}</h1>
+</md-card>
+<md-card class="default-card">
+  <md-hint class="error-hint">
```

Comment:
Only the last error is shown here, but what's in the case that a user types in multiple email adresses which already exist or are no valid email adresses at all? Please improve that by
a) removing invalid/existing email adresses before showing an error message such as ".@." is no valid email adress and was removed." and the same for existing email addresses
b) showing all messages for all errors occurred during entering the email addresses